ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,467 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+
+import rdflib
+from rdflib import URIRef
```

Comment:
We usually  avoid importing specific functions or classe, and prefer using the long version, .e.g.,
use ``` rdflib.Graph()`` below. The reason for this is the reader can tell immediately where a particular class is coming from, and doesn't have to search back to the header to see where the class was imported from.